configs/teller/config.toml.example

```diff
@@ -1,6 +1,8 @@
 cpu = 1
 gRPCPort = 50051
 testnet = true
+internal = false
+certFile = "" # Set the certificate path (e.g., "/root/certs/tls.crt") if using an internal connection (internal = true)
```

Comment:
I guess you can put the e.g. part in the brackets and put the comment after, since it's an example file anywasy.